### ISSUE TITLE
Forgotten Circles #113 - completely frozen after hitting undo, can't access menu any longer on IOS, PC, Android

### ISSUE THREAD

**Issue by itemforty** [Lurkars/gloomhavensecretariat#143]
I can't access the main menu in order to share any code, but if you can access the scenario with our password I'll share in a DM.

We started playing Forgotten Circles Scenario 113, started the round, and I accidentally moved a monster's initiative order. I then went to the main menu to hit "undo" and everything froze for myself and 4 other devices. We then reloaded the web app and I tried to "undo" again, same issue. After another reload, none of our changes would display for each other. Now near a PC I'm having the same issue trying to access the main menu. I *can* restart the scenario from the bottom "scenario" area however. 

Our setup is 5 Android devices, connecting to the public network, 4 of which are used by each player, with the last one being a device we Chromecast to a TV. Also I don't know if it helps, but we noticed the pinch to zoom you added recently, thought it was awesome, and messed around with that a bit before we started the round.

**Comment by Lurkars** [comment]
Are you able to click on feedback, report issue, download all and send me those files? Very hard to analyze what happened from this side now. But must definetly corrupted state and therefore a bug!

**Comment by itemforty** [comment]
Is there a direct link to the feedback area? Hitting the main menu icon freezes everything up. Is there a way to download all local storage, or are you ok with copying and pasting the different key values?

**Comment by Lurkars** [comment]
Just copy key values, if sensitive, send to issue@gloomhaven-secretariat.de

**Comment by itemforty** [comment]
Randomly the desktop version isn't acting up any longer, could download from Feedback:

[issue 143 - forgotten circles 113.zip](https://github.com/Lurkars/gloomhavensecretariat/files/10841596/issue.143.-.forgotten.circles.113.zip)

**Comment by itemforty** [comment]
Ok, after killing and reloading the page I noticed I could make a change on any figure, which made the menu is accessible again. I can get it to freeze up again if I "undo" enough changes to essentially make the round start over.

As a test, I clicked all the portraits, **starting with the Berzerker**, and clicked "Next Round," and then clicked Undo until it freezed, on the Undo for "End turn for Berzerker."

![image](https://user-images.githubusercontent.com/175024/221622929-887e8930-78aa-47b4-aa17-0a34635d8db6.png)

Then I did the same, **starting with the Soothsinger**, then clicked Undo until it freezed, where it freezed on the Undo for "Start turn for Soothsinger."

I then did it **again for Naaret**, which had the same result.

**Comment by Lurkars** [comment]
Thanks for the report, I will take a deep look later!

**Comment by Lurkars** [comment]
I am really sorry, but with your data I cannot reproduce. Everything is working fine. In the mean time the server state changed of course, so I disabled sync and played around. But no errors given. So are you still able to reproduce this? And if so, maybe open Dev Console and looking for error output?

**Comment by itemforty** [comment]
Yeah I can still reproduce on android and chrome desktop. If I reload the chrome page or the mobile web-app and click the menu button it will immediately freeze. If I reload either and make a change (tap to start/end turn, add condition, etc) it works just fine, so that's a good enough workaround, but if I UNDO everything up to that very first change then it freezes again. 

Here's the error output I could find, lemme know if there's a better place to pull that data (images so as not to ruin formatting):

![image](https://user-images.githubusercontent.com/175024/221664862-4123aa01-a74a-45a4-813c-a1482f7af044.png)

![image](https://user-images.githubusercontent.com/175024/221665441-5e3814bb-fd68-4f58-9aa2-35440b48c6be.png)

**Comment by Lurkars** [comment]
Okay, where does this `data-statystic.net` url come from? Are you using some Browser Extension? This is definitely not from GHS! I would never include any third party URLs!

**Comment by itemforty** [comment]
Honestly no idea. We don't use PC/Chrome when we play, I only pulled it up for testing. I'm also using an iPad for testing, and the devices we use during a session are 5 Android phones: Pixel 5a, Pixel 4a, Pixel 3a, and two last gen Samsung phones. All the same issue. 

I turned off all my Browser Extensions just in case and tried again, same issue. Then I pulled up https://ghs.champonthis.de/ in Edge and it's the same there. 

I tried to make a recording in Chrome DevTools but it froze up so there goes that approach hahah
